playbook.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Playbook to install Taskmaster application
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: What is the Trello API key?
+      private: true
+
+    - name: trello_token
+      prompt: What is the Trello token?
+      private: true 
+
+  tasks:
+    - name: Check git install
+      become: yes
+      ansible.builtin.yum: 
+       name: git
+       state: present
+    - name: Install latest Python
+      become: yes
+      ansible.builtin.yum: 
+       name: python3
+       state: latest
+    - name: Install Poetry
+      ansible.builtin.shell: 
+        cmd: curl -sSL https://install.python-poetry.org | python3
+        creates: /home/ec2-user/.local/bin/poetry
+    - name: Create the application home directory and change ownership
+      become: yes
+      ansible.builtin.command:
+        cmd: mkdir -p /opt/todoapp && chown ec2-user /opt/todoapp
+        creates: /opt/todoapp
```

Comment:
This works nicely, but might be better using the file module, to avoid us having to think about edge cases for the status of the file, or having to keep extending this command as we find new specifics we want:
```
  - name: Create the application home directory and change ownership
    ansible.builtin.file:
      path: /opt/todoapp
      state: directory
      owner: ec2-user
    become: yes
```